src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/UpdateDto.java

```diff
@@ -143,6 +144,11 @@ public boolean isNoChange() {
         return noChange;
     }
 
+    public boolean isNoChangeRelevantPeriod() {
+        return !(Objects.equals(changeBORelevantPeriod, RelevantStatementsType.CHANGE_BO_RELEVANT_PERIOD) ||
+                Objects.equals(changeBeneficiaryRelevantPeriod, RelevantStatementsType.CHANGE_BENEFICIARY_RELEVANT_PERIOD) ||
+                Objects.equals(trusteeInvolvedRelevantPeriod, RelevantStatementsType.TRUSTEE_INVOLVED_RELEVANT_PERIOD));
```

Comment:
Can we use streams on this repo? My brain personally parses the logic a lot more easily:

```suggestion
        return Stream.of(
                Objects.equals(changeBORelevantPeriod, RelevantStatementsType.CHANGE_BO_RELEVANT_PERIOD),
                Objects.equals(changeBeneficiaryRelevantPeriod, RelevantStatementsType.CHANGE_BENEFICIARY_RELEVANT_PERIOD),
                Objects.equals(trusteeInvolvedRelevantPeriod, RelevantStatementsType.TRUSTEE_INVOLVED_RELEVANT_PERIOD)
        ).noneMatch(flag -> flag);
```